src/Umbraco.Core/Compose/UnattendedInstallComposer.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Umbraco.Core.Composing;
+
+namespace Umbraco.Core.Compose
+{
+    [RuntimeLevel(MinLevel = RuntimeLevel.Install, MaxLevel = RuntimeLevel.Install)]
```

Comment:
This is using `ComponentComposer` which is an `IComposer` and not an `ICoreComposer` which means it will run after all `ICoreComposer`s . I assume that most composers don't execute on install but perhaps it's safer to also make this implement ICoreComposer? 
This is all changing in netcore too because it's very confusing to have to deal with this kinda thing.